Documentation/cheatsheet.rst

```diff
@@ -2,157 +2,396 @@
+    cilium bpf lb list
 
+
+Add a new loadbalancer
 ::
 
-        # Check policy enforcement between two pods
+    cilium service update --frontend 127.0.0.1:80 \
+        --backends 127.0.0.2:90,127.0.0.3:90 \
+        --id 20 \
+        --rev 2
 
-        cilium policy trace --src-k8s-pod <namespace>:<pod.from> --dst-k8s-pod <namespace>:<pod.to>
+BPF
+---
 
-Loadbalancing
-=============
+List node tunneling mapping information
+::
+
+    cilium bpf tunnel list
 
+Checking logs for verifier issue
+::
+
+    journalctl -u cilium | grep -B20 -F10 Verifier
 
+List connection tracking entries:
 ::
 
-        # Get list of loadbalancer services
+    sudo cilium bpf ct list global
 
-        cilium service list
+Flush connection tracking entries:
+::
 
-BPF
-===
+    sudo cilium bpf ct flush
 
+List proxy configuration:
 ::
 
-        # List node tunneling mapping information
+    sudo cilium bpf proxy list
+
+
+Kubernetes examples:
+=====================
+
+If you running Cilium on top of Kubernetes you may also want a way to list all
+cilium endpoints or policies from a single Kubectl commands. Cilium by using
+`Kubernetes Resource Definitions
+<https://kubernetes.io/docs/concepts/api-extension/custom-resources/>`_ provides
+all this information to the user:
+
+
+Policies
+---------
+
+In Kubernetes you can use two kind of policies, Kubernetes Network Policies or
+cilium network polcies. Both can be retrieved from the ``kubectl`` command:
 
-        cilium bpf tunnel list
+.. code-block:: bash
+   :name: Kubernetes Network Policies
+   :caption: Kubernetes Network Policies
+
+    kubectl get netpol
+
+.. code-block:: bash
+   :name: Kubernetes Cilium Policies
+   :caption: Kubernetes Cilium Policies
+
+    $ kubectl get cnp
+    NAME      AGE
+    rule1     3m
+    $ kubectl get cnp rule1
+    NAME      AGE
+    rule1     3m
+    $ kubectl get cnp rule1 -o json
+
+
+Endpoints
+----------
+
+To retrieve a list of all endpoints managed by cilium, a ``Cilum Endpoint``
+command can be used.
 
 ::
 
-        # Checking logs for verifier issue
+    $ kubectl get cep
+    NAME                AGE
+    34e299f0-b25c2fef   41s
+    34e299f0-dd86986c   42s
+    4d088f48-83e4f98d   2m
+    4d088f48-d04ab55f   2m
+    5c6211b5-9217a4d1   1m
+    5c6211b5-dccc3d24   1m
+    700e0976-6cb50b02   3m
+    700e0976-afd3a30c   3m
+    78092a35-4874ed16   1m
+    78092a35-4b08b92b   1m
+    9b74f61f-14571299   7s
+    9b74f61f-f9a96f4a   7s
+
+    $ kubectl get cep 700e0976-6cb50b02 -o json
+
+    $ kubectl get cep -o jsonpath='{range .items[*]}{@.status.id}{"="}{@.status.status.policy.spec.policy-enabled}{"\n"}{end}'
+    30391=ingress
+    5766=ingress
+    51796=none
+    40355=none
+
+
+Microscope
+----------
+
+Cilium also provides an option to monitor all connections from all Kubernetes
+nodes. `Microscope <https://github.com/cilium/microscope>`_ is a distributed monitor that connects to all Cilium instances
+and retrieve monitor information from there.
```

Comment:
retrieve -> retrieves

Reply:
I would rephrase this:
Cilium also provides the ability to monitor all cilium-managed connections in the kubernetes cluster via `Microscope <https://github.com/cilium/microscope>`_. It is a distributed monitor that connects to all Cilium instances and retrieves monitor information from each node.